recipe/meta.yaml

```diff
@@ -1,16 +1,16 @@
-{% set version = "5.1.3" %}
+{% set version = "5.1.4" %}
 
 package:
   name: ipykernel
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/i/ipykernel/ipykernel-{{ version }}.tar.gz
-  sha256: b368ad13edb71fa2db367a01e755a925d7f75ed5e09fbd3f06c85e7a8ef108a8
+  sha256: 7f1f01df22f1229c8879501057877ccaf92a3b01c1d00db708aad5003e5f9238
 
 build:
   number: 0
-  skip: True  # [py<34]
+  noarch: python
```

Comment:
Can we in fact go `noarch` when the `kernel.json` still has a platform-specific path? I've wondered if we couldn't go to `file:///` refs, which are more portable... I may tinker with the bot PR...